lib/logstash/plugin_mixins/http_client.rb

```diff
@@ -89,6 +100,12 @@ def client_config
         c[:ssl].merge!(truststore_password: @truststore_password)
```

Comment:
With `:validate => :password`, you'll want to use `@truststore_password.value` here.